web/themes/new_weather_theme/assets/js/charts/WeatherChart.js

```diff
@@ -30,5 +30,68 @@ const drawChart = (container, config) => {
   return new Chart(canvas, config);
 };
 
-export { drawChart };
-export default { drawChart };
+const setupScrollButtons = (container) => {
+  const isSynced = container.dataset.syncScrolling === "true";
+  const wrapper = container.closest('.wx-chart-wrapper');
+  const left = wrapper.closest('.wx-chart-wrapper').querySelector('.wx-scroll-button[data-direction="left"]');
```

Comment:
```suggestion
  const left = wrapper.querySelector('.wx-scroll-button[data-direction="left"]');
```
Shouldn't need the `.closest()` here, right? It should just match `wrapper` again? The right arrow one doesn't have it and seems to work fine.

Reply:
Ah yeah that's a mistake, good eye